src/main/cpp/bindings.cpp

```diff
@@ -1704,22 +1704,21 @@ JNIEXPORT jstring JNICALL Java_org_osgeo_proj_SharedPointer_format
  * @param  format    The `ReferencingFormat` instance used for formatting.
  * @param  warnings  The warnings, or an empty list if none.
  */
-void send_warnings(JNIEnv *env, jobject format, const std::vector<std::string>& warnings) {
+void send_warnings(JNIEnv *env, jobject format, const std::vector<std::string>& warnings, bool isGrammarErrors = false) {
```

Comment:
We may need to replace the `bool isGrammarErrors = false` argument by `jboolean isGrammarErrors = JNI_FALSE`, and when this method is invoked, replace `true` by `JNI_TRUE`. The reason is that this argument is passed to a JNI method call, and JNI methods expect arguments in `j…`.